torch/lib/c10d/HashStore.cpp

```diff
@@ -0,0 +1,84 @@
+#include <c10d/HashStore.hpp>
+
+#include <stdint.h>
+#include <unistd.h>
+
+#include <errno.h>
```

Comment:
Pedantic: can you group C stdlib headers and C++ stdlib headers separately?

Reply:
sure, done :)